sentry_trello/plugin.py

```diff
@@ -222,8 +223,12 @@ def create_issue(self, request, group, form_data, **kwargs):
         return '%s/%s' % (card['id'], card['url'])
 
     def get_config(self, project, **kwargs):
-        self.client_errors = []
-        key_value = self.get_option('key', project)
+        def get_from_initial(initial, field):
```

Comment:
The only real suggestion here is it'd make sense to pull this up to a class method instead of nesting a function here.

Then you can use `self.get_from_initial(initial, key)` instead.

Reply:
Hmm... Well, I only use it in that particular method and I get the project field for free if inside that method.

Reply:
That's fair. I wasn't blocking, was just a suggestion. So feel free to keep it as is.